e2e/tests/components/timeline/timeline.e2e-spec.ts

```diff
@@ -0,0 +1,68 @@
+import { browser, Key } from 'protractor';
+import { TimelinePage } from './timeline.po.spec';
+import { Constants, Functions } from '../common/common.spec';
+import * as moment from 'moment-timezone';

```

Comment:
Moment is quite a bulky library. Importing this will likely add ~500kb to our bundle, slowing down the tests, could we consider just using a some text for the badge title instead of a date then we don't have to import moment?